.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: generic
+sudo: required
+dist: trusty
+
+services:
+- docker
+
+env:
+- BUILDER_NAME="ovs-cni/${TRAVIS_JOB_ID}builder"
+
+install:
+ - git reset --hard
```

Comment:
Why do we need this?

Reply:
Use the kubevirt as a base file
https://github.com/kubevirt/kubevirt/blob/master/.travis.yml#L11

Reply:
We should understand *why* first. Same for the empty "file" section.